index.html

```diff
@@ -14082,34 +14084,24 @@ <h4 id="PrimaryGlobal">4.3.6. [Global] and [PrimaryGlobal]</h4>
             </li>
           </ol>
           <p>
-            Note that ECMAScript objects need not have
-            properties corresponding to <a class="dfnref" href="#dfn-constant">constants</a>
-            on them to be considered as <a class="dfnref" href="#dfn-user-object">user objects</a>
-            implementing <a class="dfnref" href="#dfn-interface">interfaces</a> that happen
-            to have constants declared on them.
-          </p>
-          <p>
-            The value of a <a class="dfnref" href="#dfn-user-object">user object</a>’s
-            <a class="dfnref" href="#dfn-attribute">attribute</a> is retrieved using the
-            following algorithm:
+            To <dfn data-export="">get a user object's attribute value</dfn>, given a <a class="dfnref" href="#idl-interface">callback interface type</a> value <var>object</var> and attribute name <var>attributeName</var>, perform the following steps:
           </p>
           <ol class="algorithm">
             <li>Try running the following steps:
               <ol>
-                <li>Let <var>V</var> be the IDL <a class="dfnref" href="#idl-interface">callback interface type</a> value
-                  that represents the <a class="dfnref" href="#dfn-user-object">user object</a> implementing the <a class="dfnref" href="#dfn-interface">interface</a>.</li>
-                <li>Let <var>O</var> be the ECMAScript object corresponding to <var>V</var>.</li>
-                <li>Let <var>P</var> be the <a class="dfnref" href="#dfn-identifier">identifier</a> of the attribute.</li>
-                <li>Let <var>script</var> be the <a class="dfnref" href="#dfn-callback-context">callback context</a> associated with <var>V</var>.</li>
-                <li>Push <var>script</var> on to the <a class="dfnref external" href="http://www.whatwg.org/specs/web-apps/current-work/multipage/webappapis.html">stack of incumbent scripts</a>. <a href="#ref-HTML">[HTML]</a></li>
+                <li>Let <var>O</var> be the ECMAScript object corresponding to <var>object</var>.</li>
+                <li>Let <var>execution context</var> be <var>object</var>'s <a class="dfnref" href="#dfn-callback-context">callback context</a>'s <a class="dfnref external" href="https://html.spec.whatwg.org/multipage/webappapis.html#settings-object">settings object</a>'s <a class="external" href="https://html.spec.whatwg.org/multipage/webappapis.html#realm-execution-context">realm execution context</a>.</li>
+                <li>Increment <var>execution context</var>'s <a class="external" href="https://html.spec.whatwg.org/multipage/webappapis.html#entrance-counter">entrance counter</a> by one.</li>
```

Comment:
So the problem here is I don't know what to set up as the entry settings. There's no obvious function to get a realm from. Should we assume every object has a realm, and use that? Not sure what to do here.